source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -10,40 +10,35 @@ Decommission Server Pools
    :local:
    :depth: 1
 
-MinIO supports decommissioning and removing a :ref:`server pools <minio-intro-server-pool>` from a deployment. 
+MinIO supports decommissioning and removing :ref:`server pools <minio-intro-server-pool>` from a deployment with two or more pools.
+To decommission, there must be at least one remaining pool with sufficient available space to receive the objects from the decommissioned pools.
+
+Starting with ``RELEASE.2023-01-18T04-36-38Z``, MinIO supports queueing  :ref:`multiple pools <minio-decommission-multiple-pools>` in a single decommission command.
+Each listed pool immediately enters a read-only status, but draining occurs one pool at a time.
+
 Decommissioning is designed for removing an older server pool whose hardware is no longer sufficient or performant compared to the pools in the deployment. 
 MinIO automatically migrates data from the decommissioned pools to the remaining pools in the deployment based on the ratio of free space available in each pool.
 
 During the decommissioning process, MinIO routes read operations (e.g. ``GET``, ``LIST``, ``HEAD``) normally. 
 MinIO routes write operations (e.g. ``PUT``, versioned ``DELETE``) to the remaining "active" pools in the deployment.
 Versioned objects maintain their ordering throughout the migration process.
 
-.. versionchanged:: RELEASE.2023-01-18T04-36-38Z
-
-   MinIO supports :ref:`decommissioning multiple pools <minio-decommission-multiple-pools>` in one command.
-
 The procedures on this page decommission and remove one or more server pools from a :ref:`distributed <deploy-minio-distributed>` MinIO deployment with *at least* two server pools.
 
 .. admonition:: Decommissioning is Permanent
    :class: important
 
-   Once MinIO begins decommissioning a pool, it marks that pool as *permanently*
-   inactive ("draining"). Cancelling or otherwise interrupting the 
-   decommissioning procedure does **not** restore the pool to an active
-   state.
+   Once MinIO begins decommissioning a pool, it marks that pool as *permanently* inactive ("draining"). 
+   Cancelling or otherwise interrupting the  decommissioning procedure does **not** restore the pool to an active state. 
+   Use extra caution if queueing decommissioning for multiple pools.
```

Comment:
```suggestion
Use extra caution if queuing multiple pools for decommissioning
```

Something about 'queuing decommissioning' feels off. Not sure why though

Reply:
Going with
> Use extra caution when decommissioning multiple pools.